website/content/docs/secrets/pki/troubleshooting-acme.mdx

```diff
@@ -0,0 +1,198 @@
+
+When attempting to contact such a node with an ACME client, a server internal
+error will appear with the above message as well.
+
+### Resolution
+
+For each Performance Replication cluster, read the value of `/config/cluster`
+and ensure the `path` field is set. When it is missing, update the URL to
+point to this mount's path on a TLS-enabled address for this PR cluster; this
+domain may be a load balanced or a DNS round robin address. For example:
+
+```
+$ vault write pki/config/cluster path=https://cluster-b.vault.example.com/v1/pki
+```
+
+Once this is done, re-read the ACME configuration and make sure no warnings
+appear:
+
+```
+$ vault read pki/config/acme
+```
+
+## Debugging account registration failures
+
+If a server has been updated to require `eab_policy=always-required` in the
+[ACME configuration](/vault/api-docs/secret/pki#set-acme-configuration),
+new account registration (and reuse of existing accounts will fail).
+
+### Symptoms
+
+When registering a new account without an [External Account Binding
+(EAB)](/vault/api-docs/secret/pki#acme-external-account-bindings), the
+Vault Server will reject the request with a response like:
+
+> Unable to register an account with ACME server
+
+with further information provided in the debug logs (in the case of
+`certbot`):
+
+> Server requires external account binding.
+
+or, if the client incorrectly contacted the server, an error like:
+
+> The request must include a value for the 'externalAccountBinding' field
+
+In either case, a new account needs to be created with an EAB token created
+by Vault.
+
+### Resolution
+
+Using a Vault token, [fetch a new external account
+binding](/vault/api-docs/secret/pki#get-acme-eab-binding-token) for
+the [desired directory](/vault/api-docs/v1.14.x/secret/pki#acme-directories):
+
+```
+$ vault write -f pki/roles/my-role-name/acme/new-eab
+...
+directory roles/my-role-name/acme/directory
+id        bc8088d9-3816-5177-ae8e-d8393265f7dd
+key       MHcCAQE... additional data elided ...
+...
+```
+
+Then pass these fields into the ACME client. For example, with `certbot`:
+
+```
+$ certbot [... additional parameters ...] \
+    --server https://cluster-b.vault.example.com/v1/pki/roles/my-role-name/acme/directory \
+    --eab-kid bc8088d9-3816-5177-ae8e-d8393265f7dd \
+    --eab-hmac-key MHcCAQE... additional data elided ...
+```
+
+Ensure that the ACME directory passed to the ACME client matches that
+fetched from the Vault.
+
+## Debugging EAB directory bindings
+
+If an EAB account token is incorrectly used with the wrong directory, the
+ACME server will reject the request with an error about insufficient
+permissions.
+
+### Symptoms
+
+When initializing a new account against this Vault server, an error might
+appear like:
+
+> The client lacks sufficient authorization :: failed to verify eab
+
+This is caused by requesting an EAB from a directory not matching the
+one the client used.
```

Comment:
```suggestion
When initializing a new account against your Vault server, you see the error:

> The client lacks sufficient authorization :: failed to verify eab

### Cause

If you request an EAB account token from a directory different from the
directory used by the ACME client, the ACME server rejects the request because
the client lacks sufficient permission to access the token.
```
Style issue: active voice
Note: I'm not sure I fully understood which client/server is involved with each step so the suggested re-write my not be technically accurate.